pom.xml

```diff
@@ -26,13 +26,38 @@
   <properties>
     <java.version>11</java.version>
     <folio-spring-base-version>1.0.4</folio-spring-base-version>
+    <edge-api-utils-version>1.0.0-SNAPSHOT</edge-api-utils-version>
+    <wiremock-version>2.27.2</wiremock-version>
+    <sonar.exclusions>src/main/java/org/folio/edgecommonspring/domain/**, src/main/java/org/folio/edgecommonspring/client/AuthnClient.java</sonar.exclusions>
```

Comment:
Can you please add a comment explaining why these are excluded?

Reply:
AuthClient is an interface, and only entities in domain repo, I don`t think we can cover it with tests somehow

Reply:
that's fine. can you please leave a comment in the pom explaining that so it's clear those that stumble upon this later?

Reply:
added